### PR TITLE
adding rds alert conditions

### PR DESCRIPTION
**Pull request by parthw**
Adding conditions for RDS instances -  
condition 1 - is_replica  
condition 2 - is_disk_gp2  
condition 3 - is aurora

### CODE REVIEW

File: legend/metrics_library/metrics/mysql_56_rds_metrics.j2 [hunk 1]
```diff
@@ -170,13 +178,17 @@ panels:
     description: The percentage of Burst Balance remaining
     targets:
       {% for dimension in data %}
+        {% if dimension.get('is_disk_gp2') is none or dimension.get('is_disk_gp2') is sameas true %}
```
Comment (vishesh92) [inline legend/metrics_library/metrics/mysql_56_rds_metrics.j2:181]: Would this work as well? `dimension.get('is_disk_gp2', True)`

Comment (parthw) [reply legend/metrics_library/metrics/mysql_56_rds_metrics.j2:181]: jinja2 does not allow default values in get. 
Also, dimension.get('is_disk_gp2') is none is added for backward compatibility.

Comment (vishesh92) [reply legend/metrics_library/metrics/mysql_56_rds_metrics.j2:181]: Can we set a default in schema?

Comment (parthw) [reply legend/metrics_library/metrics/mysql_56_rds_metrics.j2]: done, added dimension.get('is_disk_gp2', true)

It is working as expected

**Review by vishesh92 — APPROVED** [review]

**Pull request merged**